skyfield/framelib.py

```diff
@@ -149,6 +149,34 @@ def __init__(self, doc, matrix):
     def rotation_at(self, t):
         return self._matrix
 
+class LVLH(object):
+    """The Local Vertical Local Horizontal (LVLH) reference frame.
+
+    A reference frame for a body in orbit, according to CCSDS conventions.
+    The frame origin is centered on the body `position`, with z-axis in the
+    direction of the `position.center` (i.e., the negative position vector),
+    and y-axis opposite the orbital momentum vector.
+    """
+    def __init__(self, satellite):
+        self.satellite = satellite
+        self.__doc__ = (
+            "Center ({0}) Pointing Local Vertical Local Horizontal"
+            " reference frame."
+            ).format(satellite.center)
+
+    def rotation_at(self, t):
+        position = self.satellite.at(t)
+        position_vec = position.position.km
+        velocity_vec = position.velocity.km_per_s
+        z_lvlh = -position_vec / length_of(position_vec)
+        L_vec = cross(velocity_vec, z_lvlh)
+        y_lvlh = -L_vec / length_of(L_vec)
+        x_lvlh = cross(y_lvlh, z_lvlh)
+
+        matrix = array([x_lvlh, y_lvlh, z_lvlh]).T
```

Comment:
Thanks for reaching out, the test data that you mentioned would be excellent! I had been meaning to spend some time thinking about additional testing strategies, so any data that you could provide would be very much appreciated.
Are you able to attach the data here? Otherwise, could you email it through to - james dot grimmett at protonmail dot com? 

I think you make a good point about scipy Rotation objects being a nice, versatile way to carry the rotation information. I would lean toward keeping the return value as a matrix, however. Mainly to remain consistent with several other `rotation_at` functions scattered throughout the `skyfield.framelib` module, and also to avoid adding a `scipy` dependency. That said, I would be curious what @brandon-rhodes opinion is - it may be an option that has been considered before?

Reply:
@jamesgrimmett — I agree entirely. SciPy is even larger than NumPy, and it would be a shame to nearly double the install size of Skyfield just for a single feature by adding it as a dependency. The `Rotation` object also strikes me as a bit expensive, in terms of its setup cost (as I'm reading through its `from_matrix()` builder method), for a basic operation in Skyfield.

If there's a good use case for turning Skyfield rotations into `Rotation` objects, let's just plan on showing in the documentation how it would work, so that folks who need SciPy can import it into their own code and build the `Rotation`.